TheWarWithin/DruidRestoration.lua

```diff
@@ -276,40 +284,50 @@ spec:RegisterAuras( {
     harmony_of_the_grove = {
         id = 428737,
         duration = 15,
+        max_stack = 3
+    },
+    -- The actual incarn buff
+    incarnation = {
+        id = 117679,
+        duration = 3600,
         max_stack = 1
     },
+    -- This is the form
     incarnation_tree_of_life = {
         id = 33891,
         duration = 30,
```

Comment:
Fix duration.  incarnation and incarnation_tree_of_life should be same aura; this can be tree_of_life / tree_of_life_form.

Reply:
They are distinct in-game and the registration name is their real name in-game.

```
player_buffs:
     1126 - mark_of_the_wild                         -   1 - 2446.09
    33891 - incarnation_tree_of_life                 -   1 - 3600.00
...
...
...
...
   117679 - incarnation                              -   1 - 28.64 
```

Do you still want to rename the form one? or just leave as copy name for use in the code?

![Screenshot 2024-11-11 215646](https://github.com/user-attachments/assets/341ccf09-1f3f-4133-a81d-daadca0b3af1)


![image](https://github.com/user-attachments/assets/aab4a128-6bfa-484e-9d51-35aa7d4403e4)

Reply:
No, I didn't realize the aura design in-game was dumb.  The critical piece is that the tokens match SimC and any priority logic.